plugins/Kaleidoscope-FocusSerial/src/kaleidoscope/plugin/FocusSerial.cpp

```diff
@@ -77,18 +77,27 @@ EventHandlerResult FocusSerial::afterEachCycle() {
   return EventHandlerResult::OK;
 }
 
+void ledModeHandler_(const __FlashStringHelper *name) {
+  Runtime.serialPort().println(name);
+}
```

Comment:
Shouldn't this return `EventHandlerResult::OK`, similar to `::Focus.sendName`?
Even if it doesn't, with the current signature of `onLedEffectQuery` we might as well rename `::Focus.sendName` to `::Focus.sendLine` and use that, rather than having a handler method that does almost exactly the same :)

Reply:
EventHandlerResult::OK is something that gets returned by the actual EventHandler, which, in this case, is `onLedEffectQuery` 

This is made more confusing than it should be by me misnaming the callback that prints out the LED mode `ledModeHandler_` rather than `sendLedModeCallback_` or something like that.

And I think I agree more with your comment down-conversation about moving the 'unnamed mode' guard into the callback than about reusing sendName as the callback. 

I didn't realize that with the signature `const __FlashStringHelper *name` we could *also* pass in regular in-memory strings and that everything would work.)

(I do think sendName should get refactored, but let's tackle that outside this PR)

Reply:
> This is made more confusing than it should be by me misnaming the callback that prints out the LED mode `ledModeHandler_` rather than `sendLedModeCallback_` or something like that.

I agree. I'll rename the method